### PR TITLE
gh-119357: Increase test coverage for keymap in _pyrepl

### PR DESCRIPTION
**Pull request by eugenetriguba**
This PR adds a few test cases to the keymap module tests to increase the coverage.

```
Lib/_pyrepl/keymap.py                 79      0     48      0   100%
```

* Issue: gh-119357

### CODE REVIEW

File: Lib/_pyrepl/completing_reader.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
 # types
 Command = commands.Command
 if False:
-    from .types import Callback, SimpleContextManager, KeySpec, CommandName
```
Comment (eugenetriguba) [inline Lib/_pyrepl/completing_reader.py:33]: note: these two aren't used

**Comment by eugenetriguba** [comment]
I'd think this PR can skip news

File: Lib/_pyrepl/keymap.py [hunk 2]
```diff
@@ -146,11 +150,11 @@ def _parse_key1(key, s):
                 meta = 1
                 s += 3
             elif c.isdigit():
-                n = key[s + 1 : s + 4]
+                n = key[s + 1: s + 4]
```
Comment (ambv) [inline Lib/_pyrepl/keymap.py]: that's misformatted. Those are not annotations. It was good before. Ditto all other slices you changed.

Comment (eugenetriguba) [reply Lib/_pyrepl/keymap.py]: Will fix, thanks 🙂

**Comment by eugenetriguba** [comment]
@ambv beat me to it, thanks for the fix 🙂 I'll update the issue PR to more clearly encompass all of PyREPL instead of just the keymap module. Thanks for the feedback, let me know if you had any other requested changes!

**Pull request merged**